### PR TITLE
[MIG]stock operating unit to v10

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Stock with Operating Units
==========================

This module introduces the following features:
- Adds the operating unit to the Warehouse.
- Adds the operating unit to the Stock Location.
- Adds the requesting operating unit to stock pickings.
- Implements user's security access rules.

Configuration
=============

To configure this module, you need to:

* Assign Operating Unit to Warehouses.
* Assign Operating Unit to Stock Locations.

Usage
=====

This module defines the operating unit entity and the user's security rules.
Other modules extend the standard Odoo apps with the OU.

# Depends on https://github.com/OCA/operating-unit/pull/50

### CODE REVIEW

**Comment by lreficent** [comment]
Depends on https://github.com/OCA/operating-unit/pull/50 as well.

**Comment by AaronHForgeFlow** [comment]
cc @jbeficent @darshan-serpent @mreficent @guewen @lreficent

File: stock_operating_unit/demo/stock_demo.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    </record>
+
+    <!-- Chicago Warehouse -->
+    <record id="stock.stock_warehouse_shop0" model="stock.warehouse">
+        <field name="operating_unit_id" ref="operating_unit_shop0"/>
+    </record>
+
+    <!-- Your Company, Chicago shop -->
+    <record id="stock.stock_location_shop0" model="stock.location">
+        <field name="operating_unit_id" ref="operating_unit_shop0"/>
```
Comment (AaronHForgeFlow) [inline stock_operating_unit/demo/stock_demo.xml:37]: For some reason the system is setting Main Operating Unit to Chicago warehouse. However we are specifying Chicago Operating Unit here. I don't understand what is happening. @jbeficent @lreficent @mreficent

Comment (LoisRForgeFlow) [reply stock_operating_unit/demo/stock_demo.xml:37]: The same error is happening in v9 and it is merged... It is some problem with the hook with the demo data. We'll try to fix

edit:
I have tried both in local v9 and v10. v9 works. Sorry.

Comment (LoisRForgeFlow) [reply stock_operating_unit/demo/stock_demo.xml:37]: The problems seems to be a <code>stock.warehouse</code> called 'My Company, Chicago' that is only created as demo data in v10. I didn't find where that record is created though, so don't know what is the external_id to add to it the correct  operating_unit_id when installing this module. I think that would solve the problem.

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM now :+1:

**Review by serpentcs-dev1 — APPROVED** [review]
:+1: LGTM tested and code review

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**